internal/agent/install.go

```diff
@@ -98,13 +99,17 @@ func ManualInstall(c string, options map[string]string, strictValidations bool)
 	mergeOption(configStr, options)
 
 	if options["device"] == "" {
-		options["device"] = cc.Install.Device
+		if cc.Install.Device == "" {
+			options["device"] = detectDevice()
```

Comment:
is this enough to allow us to run `kairos-agent manual-install config.yaml` (without specifying `--device`)?
I remember there is a check the makes the command fail when `--device` is not specified.

Reply:
it actually panics lol

--device is NOT a required flag, so you can pass an empty value and call it a day. This may have changed behaviour a bit, but currently is broken.